src/SignalR/clients/ts/signalr/src/IHttpConnectionOptions.ts

```diff
@@ -65,4 +65,12 @@ export interface IHttpConnectionOptions {
      * Cookies are used by many load-balancers for sticky sessions which is required when your app is deployed with multiple servers.
      */
     withCredentials?: boolean;
+
+    /**
+     * Default value is 100,000 milliseconds.
+     * Timeout to apply to Http requests.
+     *
+     * This will not apply to Long Polling poll requests, EventSource, or WebSockets.
+     */
+    defaultHttpTimeoutInMilliseconds?: number;
```

Comment:
I think we decided on `httpTimeout` for this, right?

Reply:
That or `defaultTimeout`?

Reply:
httpTimeout is better IMO, used that

Reply:
I also like `requestTimeout` since this is already on IHttpConnectionOptions, but I'm fine with it as is.